en/reference/annotations-mapping.rst

```diff
@@ -50,7 +50,9 @@ Optional attributes:
 
 -  **nodeType**: PHPCR type for this node, default ``nt:unstructured``.
 -  **repositoryClass**: Name of the repository to use for this document.
--  **versionable**: Set to true to enable versioning, implies *referenceable*.
+-  **versionable**: Set to ``"simple"`` or ``"full"`` to enable versioning 
+   (respectively simple or full level). Set to ``"false"`` to disable versioning
```

Comment:
this should be just ``false`` with the backticks but without the quote. with quotes it would be the string false rather than boolean false.